.coafile

```diff
@@ -0,0 +1,25 @@
+[java]
+tags = jetbrains
+bears = ArtisticStyleBear, CheckstyleBear, coalaBear, CPDBear, JavaPMDBear
+files = **.java
+language = Java
+checkstyle_configs = google
+use_spaces = true
+indent_size = 4
+
+[java.misc]
```

Comment:
Could just be misc? I suppose you have this to inherit the `files` parameter? Please use a generic section, you need to check the whole of files for duplicates etc, not just Java files. Consider a section named `all`.

Reply:
On it.